test/upload.html

```diff
@@ -188,7 +193,7 @@
             };
           });
 
-          upload._uploadFile(file);
+          upload._addFile(file);
```

Comment:
Why do we need to replace this method even in the tests where we don't care about `index` to be present? 
Introducing the new feature shouldn't break existing tests, as long as they are correct. 
Otherwise it feels like a breaking change to me.

Reply:
I did it just for consistency, not sure how this could break existing tests. I'll restore the original invocation, then!